App.tsx

```diff
@@ -1,15 +1,64 @@
-import { StyleSheet, Text, View } from 'react-native';
-
-import VendorsListDemo from './VendorsListDemo';
-//add this import statement
-import { getAllTestDocs } from './src/database/queries';
+import * as Font from 'expo-font';
+import * as SplashScreen from 'expo-splash-screen';
+import React, { useCallback, useEffect, useState } from 'react';
+import { StyleSheet, View } from 'react-native';
 import { H1Heading } from './assets/Fonts';
+import { getAllTestDocs } from './src/database/queries';
+import VendorsListDemo from './VendorsListDemo';
+
+// Keep the splash screen visible while we fetch resources
+SplashScreen.preventAutoHideAsync();
+
+async function loadResourcesAsync() {
+  await Promise.all([
+    // Pre-load other resources (i.e. images) here
+    Font.loadAsync({
+      'manrope-bold': require('./assets/Manrope/static/Manrope-Bold.ttf'),
+      'manrope-extraBold': require('./assets/Manrope/static/Manrope-ExtraBold.ttf'),
+      'manrope-extraLight': require('./assets/Manrope/static/Manrope-ExtraLight.ttf'),
```

Comment:
note for later: we should eventually remove these 2 imports bc we don't actually use them, and we want to minimize the size of our assets.